src/main/java/de/hysky/skyblocker/skyblock/dwarven/CrystalsHudWidget.java

```diff
@@ -152,7 +136,12 @@ public void renderWidget(DrawContext context, int mouseX, int mouseY, float delt
 	}
 
 	@Override
-	public Text getDisplayName() {
-		return Text.of("Crystals HUD");
+	protected void renderWidgetConfig(DrawContext context, float delta) {
+		renderWidget(context, delta);
+	}
+
+	@Override
+	public @NotNull Information getInformation() {
+		return new Information("hud_crystals", Text.literal("Crystals HUD"), l -> l == Location.CRYSTAL_HOLLOWS); // TODO translatable
```

Comment:
This can use the same translation key as the Config group ("skyblocker.config.mining.crystalsHud") - the current text is slightly misleading.